examples/chicago_taxi/preprocess_flink.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -u
+
+echo Starting distributed TFT preprocessing...
+
+
+DATA_DIR=$(pwd)/data
+OUTPUT_DIR=$DATA_DIR
+SCHEMA_PATH=$DATA_DIR/flink_tfdv_output/schema.pbtxt
+
```

Comment:
Do we need to specify the Schema_path here ? 

The flow would be for users to run the tfdv_analyse_validate_flink, and the output is stored in these environment variables, right ?

Reply:
Yes, SCHEMA_PATH is needed as its passed to the job. 
The user does not have to change this value.